test/jdk/jdk/jfr/event/gc/objectcount/ObjectCountEvent.java

```diff
@@ -27,11 +27,12 @@ public static void test(String gcName) throws Exception {
         recording.start();
 
         Path tempFile = Path.of("temp.jfr");
+        // if temp file exists, assertion
```

Comment:
Add the assertion please!

`Asserts.assertFalse(Files.exists(tempfile), "File already exists.");` or something similar.